### PR TITLE
thanks for the help shawn :)

### PR DESCRIPTION
**Pull request by gordenstein72**
We got the functions set up to handle the click but the maze never gets updated. We were stumped on why the populateMaze function doesn't get hit after the space is clicked. 

CODE REVIEW needed

### CODE REVIEW

File: src/actions/navbarActions.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { DropdownProps, ButtonProps } from "semantic-ui-react";
+import { emptyStatement } from "@babel/types";
```
Comment (uptonm) [inline src/actions/navbarActions.ts]: what does this do?

File: src/components/Maze/Maze.tsx [hunk 2]
```diff
@@ -13,63 +13,74 @@ import './Maze.scss';
...
+    // controls.maxAzimuthAngle = (Math.PI / 360) * angle;
+    // controls.minAzimuthAngle = (Math.PI / 360) * -angle;
+
+    controls.maxPolarAngle = (Math.PI / 360) * 180;
+
+    return () => {
+      controls.dispose();
+    };
+  }, [camera, gl]);
   return null;
```
Comment (uptonm) [inline src/components/Maze/Maze.tsx]: You might not have to return null here, generally the return statement in a `useEffect` function is a function that runs on the component's unmounting from the dom.  Shouldn't be an issue either way, just worth noting.

File: src/index.tsx [hunk 3]
```diff
@@ -1,19 +1,19 @@
-import * as serviceWorker from "./serviceWorker";
-import { Provider } from "react-redux";
-import ReactDOM from "react-dom";
-import * as React from "react";
+import * as serviceWorker from './serviceWorker';
```
Comment (uptonm) [inline src/index.tsx]: This isnt something that needs to be changed, but we should add a `.prettierrc` file so all of our autoformatters are identical

File: src/actions/navbarActions.ts (hunk 1 above)
Comment (ShawnToubeau) [reply src/actions/navbarActions.ts]: Might've been a bad auto import

File: src/index.tsx (hunk 3 above)
Comment (ShawnToubeau) [reply src/index.tsx]: Agreed, I think that was from me anyway 😅

File: src/components/Maze/Maze.tsx (hunk 2 above)
Comment (ShawnToubeau) [reply src/components/Maze/Maze.tsx]: Removing the return is giving `JSX element type 'void' is not a constructor function for JSX elements.` from <CameraController />

**Comment by gordenstein72** [comment]
Issue #21

**Pull request merged**